core/src/test/scala/org/apache/spark/sql/test/SharedSQLContext.scala

```diff
@@ -348,7 +348,7 @@ trait SharedSQLContext
 
     var SSLPara = "false"
     if (conf.contains("jdbc.tls_enable") && conf.get("jdbc.tls_enable").equals("true")) {
-      SSLPara = "true&verifyServerCertificate=false&requireSSL=true"
+      SSLPara = "true&verifyServerCertificate=false&requireSSL=true&enabledTLSProtocols=TLSv1.2"
```

Comment:
no 1.3?

Reply:
Thank you for your reminder. I will add it. 
And. It is a test, that does not affect really using.